src/client/components/pages/entities/links.js

```diff
@@ -42,8 +42,8 @@ function EntityLinks({entity, identifierTypes, urlPrefix}) {
 			</Col>
 			<Col md={4}>
 				<EntityIdentifiers
-					identifierSet={entity.identifierSet}
 					identifierTypes={identifierTypes}
+					identifiers={entity.identifierSet && entity.identifierSet.identifiers || []}
```

Comment:
Empty array `[]` we are passing here actually evaluates to `true` (Reference MDN [Truthy values](https://developer.mozilla.org/en-US/docs/Glossary/Truthy), [Falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)) insteading of the expected `false`, failing to render Entities _with no identifiers_.
https://github.com/bookbrainz/bookbrainz-site/blob/82781824c8b5b3abde7c93e156b5f63ec3cf190d/src/client/components/pages/entities/identifiers.js#L29-L30